go/tasks/tasks.go

```diff
@@ -90,6 +91,48 @@ func TaskExists(name string) (bool, error) {
 	return taskMatcher(name, tasks), nil
 }
 
+// Create attempts to delete a scheduled task.
```

Comment:
I think this should be `// Create attempts to create a scheduled task` 😅

Reply:
Indeed it should.